pkg/controller/operators/olm/operatorgroup.go

```diff
@@ -862,7 +878,7 @@ func (a *Operator) ensureOpGroupClusterRole(op *v1.OperatorGroup, suffix string)
 			ClusterRoleSelectors: []metav1.LabelSelector{
 				{
 					MatchLabels: map[string]string{
-						operatorGroupAggregrationKeyPrefix + suffix: op.GetName(),
+						operatorGroupAggregrationKeyPrefix + op.GetName() + "-" + suffix: "true",
```

Comment:
nit: should we have a `suffix` const that includes the `-`?

Reply:
or [`strings.Join(...)`](https://golang.org/pkg/strings/#Join)